### ISSUE TITLE
Point & Click to move to position.

### ISSUE THREAD

**Issue by Nandox7** [Denvi/Candle#135]
Hi,

It would be good to have an option to click in a given point in the visualization and have the cnc move to that point.
Main use case I have for this Is when re-using plates to see if what I'm about to cut will fit and adjust the home position accordingly.

Thanks!

**Comment by Konesuke** [comment]
Yes it is a good idea! But it is necessary also a line to define the machine max area!

**Comment by stikdragon** [comment]
This would be really useful, although to prevent accidental clicking make it so you need to either be in a one-shot mode, or holding a modifier key (ctrl, perhaps)

**Comment by Ramzezka** [comment]
+1.
I need that thing for make pcb.
Now i need to set zero level upper and start work, wait untill drill will be on needed hole and adjust zero X Y.

**Comment by cinguecento** [comment]
How to reset the point
